### PR TITLE
adds feedback link

### PR DESCRIPTION
**Pull request by mekarpeles**
Closes #5561 

### Testing

http://testing.openlibrary.org:1337/books/OL27900720M/Ender's_Game#reader-observations

### Screenshot

testing openlibrary org_1337_books_OL27900720M_Ender's_Game

testing openlibrary org_1337_books_OL27900720M_Ender's_Game (1)

### Stakeholders

@jimchamp

### CODE REVIEW

File: openlibrary/templates/observations/review_component.html [hunk 1]
```diff
@@ -29,8 +29,11 @@ <h2 class="observation-title">$_('Community Reviews') ($total_reviews)</h2>
...
     <div class="no-stats-message">$_('No community reviews have been submitted for this work.')</div>
-  $if ctx.user:
-    $:macros.ObservationsModal(work, _('+ Add your community review'), 'stats', 'stats-link')
-  $else:
-    <a class="login-link" href="/account/login?redirect=$key">$_('Log in to add your review')</a>
+  <div class="reviews-ctas">
+    $if ctx.user:
+      $:macros.ObservationsModal(work, _('+ Add your community review'), 'stats', 'stats-link')
+    $else:
+      <a class="login-link" href="/account/login?redirect=$(key)#reader-observations">$_('+ Log in to add your community review')</a>
```
Comment (mekarpeles) [inline openlibrary/templates/observations/review_component.html:36]: this also now jumps to the right section

File: static/css/components/observationStats.less [hunk 2]
```diff
@@ -59,19 +59,15 @@
...
-  margin-bottom: 10px;
-}
-
-.stats-link {
-  margin-left: 10px;
-
+.reviews-ctas {
+  display: flex;
+  margin: 0 10px;
+
```
Comment (mekarpeles) [inline static/css/components/observationStats.less:65]:
```suggestion
```

Comment (mekarpeles) [inline static/css/components/observationStats.less:65]:
```suggestion

```

**Comment by mekarpeles** [comment]
testing openlibrary org_1337_books_OL28654891M_Charlotte%27s_Web (1)
testing openlibrary org_1337_books_OL27900720M_Ender's_Game (3)

File: static/css/components/observationStats.less [hunk 3]
```diff
@@ -59,19 +59,20 @@
...
 
-.stats-link {
-  margin-left: 10px;
-
+.login-link, .stats-link {
+  margin: 0 10px;
   a {
     text-decoration: none;
-  }
+  }
```
Comment (mekarpeles) [inline static/css/components/observationStats.less:73]:
```suggestion
  }
```

**Review by jimchamp — APPROVED** [review]
lgtm

**Pull request merged**